lib/engine/game/g_18_zoo/step/buy_sell_par_shares.rb

```diff
@@ -54,7 +61,7 @@ def log_pass(entity)
             @log << "#{entity.name} declines to buy shares"
           end
 
-          def purchasable_companies(_entity)
+          def purchasable_companies?
```

Comment:
purchasable_companies? should return a boolean, the question mark always implies a boolean, so this needs to change